backend/src/routes/auth.js

```diff
@@ -26,9 +26,16 @@ async function firebaseAuthMiddleware(req, res, next) {
     return res.status(401).send({ error: 'Unauthorized' });
   }
   try {
-    const decodedToken = await admin.auth().verifyIdToken(loginToken);
```

Comment:
Rejected promise does not throw an error and result in the try block, so I had to catch the promise reject